src/applyContainerQuery.ts

```diff
@@ -17,9 +17,11 @@ export default function<P> (
     },
 
     render() {
-      const props: {[key: string]: any} = assign({ref: this.defineContainerComponent}, this.props);
+      const props: {[key: string]: any} = assign({}, this.props);
       props[propName] = this.state ? this.state.containerQuery : {};
-      return React.createElement(WrappedComponent, props as P);
+      return React.createElement('div', {
```

Comment:
>
I love the "structure of `<ol>`, i.e. the WrappedComponent won't be touched.", but want a way to address the cons.
>
For con 1, I think it is at least as confusing as "touching the internal structure of WrappedComponent". We are really not gaining or losing anything, so we should be fine.

I agree with you on this one. I actually think not touching the DOM structure of the element is a much better situation to be in 👍

For con 2, I also agree. It'll be fairly simply to allow for a custom component or HTML element. Happy to add it to this PR if you think it's worth while.